### DISCUSSION TITLE
[BUG]In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5718] [Q&A] [answered]
In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?
![switch](https://github.com/elsa-workflows/elsa-core/assets/171897414/ffdf393d-7b89-4880-bced-f44f2e6b8804)

**Comment by sfmskywalker** [comment, marked as answer]
When you click the ADD CASE button, you will see a new row which displays the expression editor. The following screen recording shows this in action:

![2024-07-05_19-24-58](https://github.com/elsa-workflows/elsa-core/assets/938393/ee0e5f04-1968-419c-8609-1a6de584ae33)

**Reply by shxylgithub** [reply]
Thank you, teacher
